packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -10,3 +15,80 @@ export function getVariationClassName( variation ) {
 	}
 	return `is-style-${ variation }`;
 }
+
+/**
+ * Makes a copy of an object without storing any references to the original object.
+ * @param {Object} object
+ * @return {Object} The cloned object.
+ */
+function cloneDeep( object ) {
```

Comment:
Most of these functions are generic enough that they should live in a more general library IMO.

Reply:
Also used here in the edit-site package here:

https://github.com/WordPress/gutenberg/blob/cde700936c358f6e99d926ca336464c4b6b098b2/packages/edit-site/src/hooks/push-changes-to-global-styles/index.js#L278-L278